### ISSUE TITLE
Wrong relative path in skip report when tested file is upper than invocation directory

### ISSUE THREAD

**Issue by CarycaKatarzyna** [pytest-dev/pytest#7076]
The problem is if tested file is upper than invocation directory . It occurs in skip report.
Example: 
path of tested file is:`/home/xyz/my_tests/test1.py`
and `pytest` is called in location: `/home/xyz/pytest`.

```bash
 xyz@ubuntu:~/pytest$ pytest -rs ../my_tests/test1.py
============================================== test session starts ===============================================
platform linux -- Python 3.7.5, pytest-5.4.1, py-1.8.1, pluggy-0.13.1
rootdir: /home/xyz
plugins: hypothesis-5.6.0
collected 1 item                                                                                                 

../my_tests/test1.py s                                                                                     [100%]

============================================ short test summary info =============================================
SKIPPED [1] my_tests/test1.py:3: no way of currently testing this
=============================================== 1 skipped in 0.01s ===============================================
```
Instead ``SKIPPED [1] my_tests/test1.py:3: no way of currently testing this`` it should be
``SKIPPED [1] ../my_tests/test1.py:3: no way of currently testing this``.

**Comment by tirkarthi** [comment]
Seems related : https://github.com/pytest-dev/pytest/issues/4677

**Comment by CarycaKatarzyna** [comment]
@tirkarthi Yes, it seems similar, but doesn't apply to the same. In [#4677](https://github.com/pytest-dev/pytest/issues/4677) we want to change absolute paths to relative. Solved #4677 will leave already relative paths unchanged and will not fix this issue.
